### ISSUE TITLE
 'Pie chart' present in 'Security posture' page is not Keyboard accessible. 

### ISSUE THREAD

**Issue by msftedad** [przpiw/Nivo-PieChart-Demo#2]
Severity: Sev2 (High)

Test Environment: OS: Windows 11 22H2
OS Build: 23620.1000

Repro steps:
1.Open the Above URL: [Pie chart | nivo](https://nivo.rocks/pie/)
2.Pie chart page will appear on the screen
3.Now try to navigate to pie chart and observe the issue whether pie chart is accessible or not.

Actual Result:
The 'Pie Chart' present in 'Pie' page is not Keyboard accessible.

Expected Result:
The 'Pie chart' present  'pie' page should be Keyboard accessible.
And Screen reader focus should move on pie chart content and announce the chart data information.

Attachments:

https://github.com/przpiw/Nivo-PieChart-Demo/assets/93735775/9ba94e92-f529-4e06-9f01-4326c904d0ef

**Comment by msftedad** [comment]
Hi Team, Any update on this issue? Please provide the ETA to fix this issue.
